App/routes/operations.py

```diff
@@ -0,0 +1,26 @@
+from flask import jsonify
+from main import app
+
+@app.route("/api/calc/sumar/<num1>/<num2>")
+def api_calc_sumar(num1, num2):
+   resultado = int(num1) + int(num2)
+   #return render_template('Suma.html', resultado = resultado, num1 = num1, num2 = num2)   
```

Comment:
remove commented code, comments should be used to add documentation to the app.

Reply:
fixed